specification/schemas/returns/ReturnItem.json

```diff
@@ -3,69 +3,17 @@
   "additionalProperties": false,
   "required": [
     "id",
-    "quantity",
-    "description",
-    "item_price"
+    "quantity"
```

Comment:
Note to self: all these removals should only have effect on ReturnItems in a POST request. Will split this schema into GetReturnItem and PostReturnItem